src/DocBlock/Tags/InvalidTag.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace phpDocumentor\Reflection\DocBlock\Tags;
+
+use phpDocumentor\Reflection\DocBlock\Tag;
+use Throwable;
+use Webmozart\Assert\Assert;
+
+/**
+ * This class represents an exception during the tag creation
+ *
+ * Since the internals of the library are relaying on the correct syntax of a docblock
+ * we cannot simply throw exceptions at all time because the exceptions will break the creation of a
+ * docklock. Just silently ignore the exceptions is not an option because the user as an issue to fix.
+ *
+ * This tag holds that error information until a using application is able to display it. The object wil just behave
+ * like any normal tag. So the normal application flow will not break.
+ */
+final class InvalidTag implements Tag
+{
+    /** @var string */
+    private $name;
+
+    /** @var string */
+    private $body;
+
+    /** @var Throwable */
+    private $throwable;
+
+    private function __construct(string $name, string $body, Throwable $throwable)
+    {
+        $this->name      = $name;
+        $this->body      = $body;
+        $this->throwable = $throwable;
+    }
+
+    public function getException() : Throwable
+    {
+        return $this->throwable;
+    }
+
+    public function getName() : string
+    {
+        return $this->name;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public static function create(string $body, string $name = '', ?Throwable $exception = null)
```

Comment:
I like how you store the exception; that way we could use this info to render an error log in phpDocumentor

Reply:
I do wonder whether it would not be better to introduce a 'withError()' method with which you can return a new version of this tag with the exception attached; this will keep the interface cleaner and will better emulate the optional character. The caller already knows it has to do something special for this tag; so for a caller it doesn't really matter.

(example: if we were to ever change the `create` definition in the interface; this would break any code that uses this exception parameter; with `withError` you avoid that problem)